dockernet/config.sh

```diff
@@ -374,6 +381,28 @@ WAIT_FOR_BALANCE_CHANGE() {
   done
 }
 
+# Sleep until the total delegation amount has changed
+# Optionally provide a minimum amount it must change by  (to ignore interest)
+WAIT_FOR_DELEGATION_CHANGE() {
+  chain_id=$1
+  minimum_change=${2:-1} # defaults to 1
+
+  max_blocks=30
+
+  $STRIDE_MAIN_CMD q stakeibc show-host-zone $chain_id
```

Comment:
Is this supposed to be going into a variable or is it going to be echo to print out?

Reply:
+1, unclear what this line is doing

Reply:
how do you guys not see how important this line is???

Reply:
Kidding! I think I accidentally copy/pasted this line as I was writing the command a few lines below. Removed